osu.Game/GameModes/Menu/ButtonSystem.cs

```diff
@@ -285,37 +290,35 @@ public override void Load()
             {
                 base.Load();
 
-                Add(logoBounceContainer = new AutoSizeContainer());
-
-                logo = new Sprite(Game.Textures.Get(@"menu-osu"))
-                {
-                    Anchor = Anchor.Centre,
-                    Origin = Anchor.Centre,
-                };
-
-                logoBounceContainer.Add(logo);
-
                 Sprite ripple = new Sprite(Game.Textures.Get(@"menu-osu"))
                 {
                     Anchor = Anchor.Centre,
                     Origin = Anchor.Centre,
                     Alpha = 0.4f
                 };
-
-                logoBounceContainer.Add(ripple);
-
                 ripple.ScaleTo(1.1f, 500);
                 ripple.FadeOut(500);
                 ripple.Loop(300);
 
-                logoBounceContainer.Add(vis = new MenuVisualisation()
+                Add(logoBounceContainer = new AutoSizeContainer()

```

Comment:
These .Load methods are called very rarely. The overhead of allocating an array is negligible. I'd say code readability / consistency is most important as long as there is not a huge and obvious downside to performance. If something ends up being slow and a hotspot (such as found with profiling, or when comparing to master branch in simple tests) then it can still be optimized. :)

I'd go with ```new Drawable[] {}``` here.